### PR TITLE
Minor rule clarifications/additions

### PR DESCRIPTION
**Pull request by therealDLondon**
## About the PR
Changed various rules to add clarifications, and add a few lines/points concerning things admins have reached a consensus on already

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: Changed and adjusted various rules, be sure to re-read the rules!

### CODE REVIEW

File: Resources/ServerInfo/Rules.txt [hunk 1]
```diff
@@ -35,9 +35,10 @@ If you have any questions about these rules, please use the admin help menu by h
 [color=#a4885c]1.[/color] Follow the server expectations:
  - If you are banned from a role or department, you may not play that role. This includes seeking or accepting promotions into roles you are banned from.
  - Do not make yourself a major problem/annoyance/disruption for the crew while not being an antagonist (i.e. self-antagging). Intentionally hindering the station as a crew-aligned character is strictly against our rules.
- - Department strikes (e.g. cargonia and any variation thereof), riots, cults, and any other type of similar largely disruptive behavior are strictly forbidden without both excellent roleplay justification and administrator pre-approval.
+ - Clowns, Mimes, and other "prankster" roles do not get exemption from the above, and should not use their job as an excuse to be a major disturbance.
```
Comment (Colin-Tel) [inline Resources/ServerInfo/Rules.txt]:
```suggestion
 - Clowns, Mimes, and other "prankster" roles do not get an exemption from the above, and should not use their job as an excuse to be a major disturbance.
```

**Review by Colin-Tel — COMMENTED** [review]
Grammar pass, I don’t necessarily agree with the rule changes themselves.

**Comment by deltanedas** [comment]
> Clowns, Mimes, and other "prankster" roles do not get an exemption from the above, and should not use their job as an excuse to be a major disturbance.

rip shitters

File: Resources/ServerInfo/Rules.txt [hunk 2]
```diff
@@ -61,39 +62,40 @@ If you have any questions about these rules, please use the admin help menu by h
...
     - Both departments are required to read and follow Delta-V Space Law, Standard Operating Procedure, Alert Procedure, and Company Policy to the best of their ability.
     - Abandoning your role to go do whatever you want instead of managing your department or maintaining security, abusing your position or using it to make arbitrary or malicious choices to the detriment of the station, and being negligent to the point of harm is prohibited.
  - If you need to leave the round, please notify administrators via AHelp menu and notify your fellow crew via departmental radio. If you have the capability to do so, ensure that you return your items as necessary and seek cryostorage if you are not planning on returning.
- - Do not give up or trade away Traitor objective items, departmental/Command gear, and sensitive equipment without excellent reason.
+ - Do not give up or trade away Traitor objective items, departmental/Command tools and remotes, and sensitive equipment without excellent reason.
```
Comment (deltanedas) [inline Resources/ServerInfo/Rules.txt:98]: departmental tools = ce isnt allowed to give crowbars to people?

Comment (therealDLondon) [reply Resources/ServerInfo/Rules.txt:98]: More like Epi shouldnt be giving away use of their protowhatever, or cargo shouldnt give away PKAs or Salvage Suits.

A crowbar is part of the maints department anyway

Comment (deltanedas) [reply Resources/ServerInfo/Rules.txt:98]: that only applies to command so scientists themselves are allowed to do it

this should just be in SOP that protolathes and circuit imprinters are controlled items so they are ic guilty of black marketeering if they give it away

Comment (therealDLondon) [reply Resources/ServerInfo/Rules.txt:98]: Its a "stopgap" until SOP is re-written anyhow. Just stops command giving away things for free for seemingly no reason. Should also help metafriending

**Pull request merged**